.github/actions/import/action.yml

```
@@ -60,19 +77,53 @@ runs:
           exit 1
         fi
         if [ -z "${{ inputs.files_path }}" ]; then
-          echo "The files_path input is cannot be an empty string"
+          echo "The files_path input cannot be an empty string"
           exit 1
         fi
         if ! [ -d "${{ inputs.files_path }}" ]; then
           echo "The files_path input '${{ inputs.files_path }}' doesn't exist, make sure the checkout action was called"
           exit 1
         fi
 
+    - name: Validate Data
+      if: ${{ !inputs.import_only }}
+      id: validate
+      shell: bash
+      env:
+        DESCOPE_MANAGEMENT_KEY: ${{ inputs.management_key }}
+        DESCOPE_BASE_URL: ${{ inputs.descope_base_url }}
+      run: |
+        echo "Validating project data"
```

```suggestion
        echo "Validating project"
```

I have a distaste from the term "data" 😅